itest/rust/src/variant_test.rs

```diff
@@ -60,6 +59,13 @@ fn variant_conversions() {
     roundtrip(2147483647i32);
     roundtrip(-2147483648i32);
     roundtrip(9223372036854775807i64);
+
+    // string
+    roundtrip(gstr("some string"));
+    roundtrip(String::from("some other string"));
+    let str_val = "abcdefghijklmnop";
+    let back = String::try_from_variant(&str_val.to_variant()).unwrap();
```

Comment:
Does `String::from_variant()` instead of `String::try_from_variant().unwrap` not work because the feature is not enabled? If yes, then leave it. I'll probably remove that feature.

Reply:
it does work actually, i just copied what `roundtrip()` did initially